CONFLICTS.md

```diff
@@ -3,6 +3,10 @@
 This document explains why certain conflicts were added to `composer.json` and
 references related issues.
 
+- `doctrine/common:3.1.2` && `doctrine/orm:2.8.3`:
+
+  doctrine/common:3.1.2 with doctrine/orm:2.8.3 introduces a series of changes that make our Behat suite fail on database layer.
```

Comment:
I'm not convinced that this is true. As I see, the static analysis is failing, not behats. What is more, the build is failing on master and probably also on other branches, so I would be for fixing that issue in a separate PR.

Reply:
Then we need someone who is able to do it. My psalm creates some memory leak and I am not able to test and check on local device.